share/spice/symbolab/symbolab.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_symbolab = function (api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.solution) {
+            return Spice.failed('symbolab');
+        }
+
+        DDG.require('mathquill', function (){
+            Spice.add({
+                id: "symbolab",
+                name: "Calculator",
+                data: api_result.solution,
+                meta: {
+                    sourceName: "Symbolab.com",
+                    sourceUrl: api_result.solution.url
+                },
+                normalize: function (data) {
+                    return {
+                        latex: data.originalQuery.replace(/ /g, "\\text{ }"),
+                        solution: data.solutionLatex.replace(/^=/, "").replace(/,/, ",\\text{ }"),
```

Comment:
It's not required to replace commas, but it looks slightly better when there are multiple results. like in case for x^2=25. We should add the /g to the regex.